infection_monkey/model/__init__.py

```diff
@@ -4,6 +4,7 @@
 
 MONKEY_ARG = "m0nk3y"
 DROPPER_ARG = "dr0pp3r"
+ID_STRING = "M0NK3Y3XPL0ITABLE"
```

Comment:
If this is unique to the struts attack, what is it doing here?

Reply:
In my case if the payload is wrong we get the whole page. To know that payload wasn't wrong and we in fact could exploit, we need to look for as unique as possible string(ID STRING). This string may be used for other purposes or other exploits that share the same problem